### PR TITLE
Update FAQ reg. luca stand-by (Addresses #2751)

### LINKED ISSUE #2056: FAQ: Mention that only luca codes generated by businesses (not the user-generated ones) are compatible with CWA
## Where to find the issue
FAQ

## Describe the issue / Suggested change
The FAQ should mention, that QR codes generated by users in their Android/iOS luca-apps ("Meinen QR-Code anzeigen" as well as "privates Treffen erstellen") use different formats that are not compatible with the CWA. Only codes generated by businesses at https://app.luca-app.de/login are compatible.

### LINKED ISSUE #2088: FAQ: #check_in_luca_error
New FAQ:
- **DE** https://www.coronawarn.app/de/faq/#check_in_luca_error 
- **EN** https://www.coronawarn.app/en/faq/#check_in_luca_error

To be covered:

- private luca QR codes
- outdated luca QR codes, generated before 25 May
- luca QR codes generated after 25 May, but not CWA compatible
- "Faulty" luca QR codes, see https://github.com/corona-warn-app/cwa-website/pull/2069

Related:
https://github.com/corona-warn-app/cwa-wishlist/issues/680
https://github.com/corona-warn-app/cwa-website/pull/2069
https://github.com/corona-warn-app/cwa-documentation/issues/741

---
Internal Tracking ID: [EXPOSUREAPP-10564](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10564)
Internal Tracking ID: [EXPOSUREAPP-10617](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-10617)

### LINKED ISSUE #2751: luca app announces it is on stand-by
## Your Question

According to https://www.luca-app.de/luca-digitale-plattform-fuer-buergerinnen/ luca has gone into stand-by.

- Are any changes necessary to the articles in the section https://www.coronawarn.app/en/faq/results/#check_in_luca?
- ~~Should the PR https://github.com/corona-warn-app/cwa-website/pull/2164 be processed or closed?~~

At least https://www.coronawarn.app/en/faq/results/#check_in_luca_why_now is outdated.

---
Internal Tracking ID: [EXPOSUREAPP-12828](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12828)
Internal Tracking ID: [EXPOSUREAPP-12850](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12850)

### PR DESCRIPTION
**Pull request by Ein-Tim**
## Description

This PR updates many FAQ articles reg. CWA & Luca compatibility, fixes an issue with English anchor links for luca FAQs, and adds a redirect from `check_in_luca_` to section `check_in_luca`, as the anchor is no longer used.

## Screenshots

Unlike the .zip file before which required a decent amount of effort for naming etc, I've found an easier way of visualizing the changes without making this opening comment too long. No download required anymore, it's just a long image uploaded on an image hoster

- [German FAQs](https://i.imgur.com/0lHGQkP.png)
- [English FAQs](https://i.imgur.com/RKdI0Al.png)

---

**Closes #2751**

This PR might also close #2088.
This PR might also close #2056.

---
Internal Tracking ID: [EXPOSUREAPP-12850](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12850)
Internal Tracking ID: [EXPOSUREAPP-12828](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12828)

### CODE REVIEW

File: src/data/faq.json
Comment (Ein-Tim) [inline src/data/faq.json:2929]: The anchor here is actually missing the word "both" at the end, however, I still included this in the redirects, to make sure that even if this incomplete anchor link floats around somewhere on the internet, a useful response is shown.

**Comment by Ein-Tim** [comment]
@dsarkar

Will you follow up and let us know whether it will continue to be possible to check-in with CWA by scanning luca QR codes of if this feature will be shut down.

File: src/data/faq.json
Comment (MikeMcC399) [reply src/data/faq.json:2929]: I'm not sure that is necessary, since this incorrect anchor `check_in_luca_` was only introduced into the English page when the luca Q&As were split up in the new FAQ design, so it hasn't been present for very long. I would ignore this error, but that is just my personal preference.

Comment (Ein-Tim) [reply src/data/faq.json:2929]: I have no strong feelings in any direction, so I would leave it to the Open-Source-Team to decide whether this should also be a redirect or not.

**Comment by dsarkar** [comment]
@Ein-Tim Currently it is not possible to check-in with CWA by scanning luca QR codes.

**Comment by Ein-Tim** [comment]
@dsarkar Attempting to scan the Luca QR code shared by @floscher in https://github.com/corona-warn-app/cwa-documentation/issues/590#issuecomment-829349393 is successful. So please, cross check this information.

**Comment by floscher** [comment]
> @Ein-Tim Currently it is not possible to check-in with CWA by scanning luca QR codes.

@dsarkar That is not true. Due to privacy concerns the CWA must never in the scanning process contact the luca servers. Therefore a shutdown of luca servers won't impact, if you can checkin with CWA or not. As long as there are no changes made by the CWA team (specifically to these lines https://github.com/corona-warn-app/cwa-server/blob/8e483dedf71ee64497c146b5887980e73714b81b/services/distribution/src/main/resources/main-config/v2/presence-tracing-parameters.yaml#L123-L127), the CWA can still scan luca codes.

And as I said in #2751 I would propose to keep the compatibility with luca for now, in order to give those who want to check in with CWA additional options.

One concern that should be kept in mind would be, that luca could from now on switch their QR codes back to an incompatible format. But any codes generated between May 2021 and April 2022 would definitely still be compatible regardless.

**Comment by MikeMcC399** [comment]
- Apart from the technical side, I suggest also to check if there are any contractual agreements in place between RKI and the owners of luca. See my https://github.com/corona-warn-app/cwa-website/issues/2751#issuecomment-1099888631 in the related issue.

**Review by Ein-Tim — COMMENTED** [review]
Little update:
Scanning the QR code in https://github.com/corona-warn-app/cwa-documentation/issues/590#issuecomment-829349393 is still successful with CWA. It does not seem like there was any change done, preventing checking in with CWA by scanning a luca QR code.

**Comment by dsarkar** [comment]
@Ein-Tim Indeed, Internal Tracking ID: [EXPOSUREAPP-12850](https://jira-ibs.wbs.net.sap/browse/EXPOSUREAPP-12850) deals with this.

**Comment by floscher** [comment]
@Ein-Tim @dsarkar Would it make sense to update the FAQ with the status quo for now (scanning luca QR codes works, but it's not clear for how long it will continue to work) and then later update the FAQ entry when a decision is made?

**Comment by dsarkar** [comment]
@floscher Thanks for the suggestion. I will rise this internally.

**Comment by Ein-Tim** [comment]
I don't think this is an urgent item as at the moment nothing changed for CWA users. I vote for waiting till there was a decision made whether it will stay possible to use luca QR codes to check in. But I might be missing something.

**Comment by floscher** [comment]
> Little update: Scanning the QR code in [corona-warn-app/cwa-documentation#590 (comment)](https://github.com/corona-warn-app/cwa-documentation/issues/590#issuecomment-829349393) is still successful with CWA. It does not seem like there was any change done, preventing checking in with CWA by scanning a luca QR code.

Another little update: The luca app is no longer able to check-in to such a QR code. Check-in with the CWA continues to work. I guess it is an oversight on luca's part, probably because the daily keypair of health authorities is now no longer accessible (the last keys were generated about a month ago on 30th of March, but are now gone).
As far as I understood they were planning to continue using the QR codes to just show the menu or the website of the place. So probably they will fix it again sooner or later …

**Comment by MikeMcC399** [comment]
It would be helpful if the app publishers (RKI and culture4life GmbH) communicated with each other and came up with a common public statement about the use of luca codes with CWA. Currently users are left uninformed.

**Comment by Ein-Tim** [comment]
Dear Open-Source-Team (@larswmh & @dsarkar),

I provided this PR nearly a month ago and since two weeks there is no process on this item anymore. Could you please decide how to proceed here (https://github.com/corona-warn-app/cwa-website/issues/2751#issuecomment-1113334921)? 

Thank you!
Tim

**Comment by dsarkar** [comment]
@Ein-Tim @MikeMcC399 We are having a look at this now.

**Comment by Ein-Tim** [comment]
@dsarkar Did the team came to a decision on how to proceed with this item already?

**Comment by Ein-Tim** [comment]
Merge conflicts resolved. This PR is now open for over 3 months. Please provide feedback on how to continue here.

**Comment by Ein-Tim** [comment]
@larswmh 

For sure you can continue! Thanks!

**Comment by Ein-Tim** [comment]
@larswmh Thanks, I will review the PR later.

**Review by Ein-Tim — COMMENTED** [review]
Some in-line comments.

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:1072]: I am quite confused by this paragraph. I suggest to delete it. It is describing unnecessary QR codes which are no longer used.

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:1086]: What does "immediately" mean here? Can this word be removed?

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:1092]: "In the case of an incompatible luca QR code, you can contact the organizer, who will issue a new Corona-Warn-App QR code if necessary."

Is that still true?

File: src/data/faq.json
Comment (MikeMcC399) [inline src/data/faq.json:2983]: "The luca app could have not scanned QR codes ..." =>
"The luca app cannot scan QR codes ..."

otherwise you have to explain about the use of the "could have not" tense

File: src/data/faq_de.json
Comment (floscher) [reply src/data/faq_de.json:980]: I think all three variants are about equal for my ears. I'd rather say that the second option is the odd one out.

How about this:
```suggestion
                                    "Anmerkung: Die luca-App bietet seit dem Frühjahr 2022 keine Eventregistrierung (Check-In-Funktionalität) mehr an. Dennoch kann mit der Corona-Warn-App durch Scannen von luca-QR-Codes weiterhin eingecheckt werden.<hr>",

```

File: src/data/faq.json
Comment (floscher) [reply src/data/faq.json:2983]: I'd suggest to just drop all these "could have not if luca still offered contact tracing". They do no longer, so this information is irrelevant for the FAQ.

**Review by MikeMcC399 — CHANGES_REQUESTED** [review]
I have not completely reviewed this PR. I started on the English text and got quite frustrated trying to understand all the conditional language in it. I made some comments in any case.

I really question whether it is helpful to users to leave all this information in the FAQs when luca abandoned the support months ago, even though they leave themselves a back-door open to returning by stating they are on "stand-by". I don't see any sign of them returning, but I'm sure I don't have the full picture.

**Comment by MikeMcC399** [comment]
I suggest a change to the general statement in English used in many different FAQs:

"Since spring 2022, the event registration (check-in functionality) of the luca app is no longer possible for the time being."

"no longer possible for the time being" is a contradictory statement since [no longer](https://dictionary.cambridge.org/dictionary/english/no-longer) is always used for a permanent condition and it does not lend itself to being modified by [for the time being](https://dictionary.cambridge.org/dictionary/english/for-the-time-being)

say "temporarily unavailable" instead

"Since spring 2022, the event registration (check-in functionality) of the luca app is temporarily unavailable." 

which would be a better translation of the German sentence:

"Die Eventregistrierung (Check-In-Funktionalität) der luca-App ist seit dem Frühjahr 2022 vorerst nicht mehr möglich."

File: src/data/faq_de.json
Comment (MikeMcC399) [reply src/data/faq_de.json:980]: @floscher 

Your suggestion is good:

Would it be clearer to say:
"Dennoch kann mit der Corona-Warn-App durch Scannen von bereits existierenden luca-QR-Codes weiterhin eingecheckt werden." ?

File: src/data/faq.json
Comment (MikeMcC399) [reply src/data/faq.json:1071]: @Ein-Tim 
Yes, it is strange. I commented about it separately in https://github.com/corona-warn-app/cwa-website/pull/2757#issuecomment-1185663618, since it is used multiple times.

The German text isn't so good either, so if this gets changed then the English text could also be completely adapted.

File: src/data/faq_de.json
Comment (floscher) [reply src/data/faq_de.json:980]: @MikeMcC399 I think it's an unnecessary qualifier, since you can only ever scan QR codes that "already exist". If they don't exist, there's nothing to scan :wink:

Also luca seems to still produce compatible QR codes by default, so even future QR codes would be compatible. I haven't checked in a while, though if that's still the case.

Comment (MikeMcC399) [reply src/data/faq_de.json:980]: @floscher 
It's certainly unnecessary to change the text if luca can still produce compatible QR codes. I was unsure about this. I was really trying to refer to previously generated QR codes on the assumption that they can't be generated any more by luca, but if that is not the case, then please disregard what I wrote! 🙂

**Comment by floscher** [comment]
I re-evaluated the compatibility of luca QR codes that are newly created. And as of today newly created luca QR codes are **not** compatible with the CWA. Previously it was the other way around, that they were compatible by default.

Now venue owners need to enable the third of these switches to make the codes compatible:
screenshot of the luca admin interface for creating QR codes

I guess we then need to add, that there is a good chance that the QR codes are compatible, but it does not work in some cases. Unfortunately there is no easy way for people to tell, which luca codes work or not, except by just trying to scan with the CWA.

**Comment by Ein-Tim** [comment]
@floscher Thank you for checking from my side!!

**Comment by GisoSchroederSAP** [comment]
The current description is correct so far, maybe we just add a remark "if the created QR code war created to be compatible with the CWA".

This is no longer an issue with our app. The luca app does not exist anymore and did not communicate any transition for their registration codes at all. So, we cannot solve this issue, but with the small update above we explain the prerquisites.

**Comment by MikeMcC399** [comment]
@GisoSchroederSAP 

> The luca app does not exist anymore and did not communicate any transition for their registration codes at all. So, we cannot solve this issue, but with the small update above we explain the prerquisites.

If your view is that the luca app does not exist any more, then I don't understand all the effort going into trying to update the FAQs in a clean way preserving a lot of detailed information and jumping through hoops.

Wouldn't it be better just to remove all the outdated information? Is RKI or the Open Source Team in contact with the responsible people for the luca app to define a communication strategy for users?

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json:3049]: Wrong anchors have now been addressed in this PR and should be fixed

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json]: Fixed in commit https://github.com/corona-warn-app/cwa-website/pull/2757/commits/0b254036e8d8d1fad6ce38c9d1614b4b9313aac2, thanks

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json]: Changed as suggested in https://github.com/corona-warn-app/cwa-website/pull/2757/commits/0b254036e8d8d1fad6ce38c9d1614b4b9313aac2, thanks

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json]: Removed that sentence in commit https://github.com/corona-warn-app/cwa-website/pull/2757/commits/0b254036e8d8d1fad6ce38c9d1614b4b9313aac2, thanks

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json]: Removed in commit https://github.com/corona-warn-app/cwa-website/pull/2757/commits/0b254036e8d8d1fad6ce38c9d1614b4b9313aac2, thanks

**Review by Ein-Tim — COMMENTED** [review]
Changes are looking good to me!

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json]: First sentence has been changed to "The luca app no longer offers event registration (check-in functionality) as of spring 2022.", as suggested by @floscher in another conversation, in all affected FAQs in https://github.com/corona-warn-app/cwa-website/pull/2757/commits/0b254036e8d8d1fad6ce38c9d1614b4b9313aac2, thanks!

File: src/data/faq.json
Comment (larswmh) [reply src/data/faq.json]: Even if no longer used, we've decided to keep this article. The remarks above the paragraph should be a signal to users that the information below might not be up to date anymore

**Comment by MikeMcC399** [comment]
@larswmh 
- Given that you are restricted in not deleting the obsolete information, I would go ahead and merge this PR. We can then close the related issue https://github.com/corona-warn-app/cwa-website/issues/2751.

**Comment by dsarkar** [comment]
@Ein-Tim @floscher @MikeMcC399 Thanks a lot for the massive effort by your suggestions and reviews you have putting into this PR.

**Pull request merged**
